app/_hub/kong-inc/application-registration/how-to/_index.md

```diff
@@ -36,6 +36,65 @@ curl -X PATCH http://localhost:8001/plugins/<plugin_id> \
   --data "config.auto_approve=true"
 ```
 
+### Allow accessing the service with consumer credentials (Optional)
+
+By enabling the `application-registration` plugin on a service, access to the service with consumer credentials will be disallowed by default. To allow this behavior, you can set `enable_proxy_with_consumer_credential` to `true`.
+
+{:.important}
+> **Important:** Please always use `enable_proxy_with_consumer_credential` with caution, as this allows accessing the scoped service via consumer credentials other than your application's client ID (API key).
+
+```
+curl -X POST http://localhost:8001/services/<service>/plugins \
+    --data "name=application-registration"  \
+    --data "config.display_name=<my_service_display_name>" \
+    --data "config.enable_proxy_with_consumer_credential=true"
+```
+
+Or, update your current configuration with a `PATCH` request.
+Replace `<plugin_id>` with the `id` from the existing application-registration instance.
+
+```
+curl -X PATCH http://localhost:8001/plugins/<plugin_id> \
+  --data "config.enable_proxy_with_consumer_credential=true"
+```
+
+And then, create a plugin `key-auth` and route for the service.
+
+```
+curl -X POST http://localhost:8001/services/<service>/routes \
+    --data "path=/test"
+```
+
+```
+curl -X POST http://localhost:8001/services/<service>/plugins \
+    --data "name=key-auth" \
+    --data "config.key_names=apikey"
+```
+
+Create a consumer and a consumer's credential for the `key-auth`.
+
+```
+curl -X POST http://localhost:8001/consumers \
+    --data "username=test"
+```
+
+```
+curl -X POST http://localhost:8001/consumers/<consumer_id>/key-auth \
+    --data "key:<apikey>"
+```
+
+When `enable_proxy_with_consumer_credential` is enabled, constructing a request to the service should yield a 200 response, rather than the 401 response when disabled.
```

Comment:
```suggestion
When `enable_proxy_with_consumer_credential` is enabled, constructing a request to the service should yield a 200 response, rather than the 401 response when disabled:
```